src/amo/components/GetFirefoxButton/index.js

```diff
@@ -0,0 +1,43 @@
+/* @flow */
+import * as React from 'react';
+import { compose } from 'redux';
+
+import { makeQueryStringWithUTM } from 'amo/utils';
+import translate from 'core/i18n/translate';
+import Button from 'ui/components/Button';
+import type { AddonType } from 'core/types/addons';
+import type { I18nType } from 'core/types/i18n';
+
+import './styles.scss';
+
+export type Props = {|
+  addon: AddonType,
+|};
+
+type InternalProps = {|
+  ...Props,
+  i18n: I18nType,
+|};
+
+export const GetFirefoxButtonBase = (props: InternalProps) => {
+  const { addon, i18n } = props;
+
+  return (
+    <Button
+      buttonType="confirm"
+      href={`https://www.mozilla.org/firefox/new/${makeQueryStringWithUTM({
+        utm_content: addon.guid,
+      })}`}
+      puffy
```

Comment:
should puffy be configurable? I just feel like we might need smaller on the guide pages (I was not using puffy on the other buttons in firefox). Maybe we can circle back tho once we know how these should look...

Reply:
I agree that all types of this button may not need `puffy`, but I'm not sure that we want to make `puffy` itself an option. It seems like rather we would need this component to support different `type`s and the component would know how to style each one. Does that make sense? 

As we don't know what the other types are yet I think we should just leave it as is for now.